### PR TITLE
Rules Update PT.2

### PR DESCRIPTION
**Pull request by GreenyMCR**

### CODE REVIEW

File: rules.json [hunk 1]
```diff
@@ -1,17 +1,170 @@
...
+    "Advertising",
+    "• Do NOT have a social media link, IP, or another FiveM server in your discord status message.",
+    "• Do NOT advertise another server in our text/voice channels. You may advertise another server in Private Direct Messages, but ONLY IF THEY ASK FOR IT. Failure to follow this rule will result in a permanent ban.",
+    "• Similarly, do not advertise your YouTube channel or Social Media to any users unless they ASK FOR IT.",
+    "Notes",
+    "Anything you say to users through private messages on any application can be punishable on the Discord Server as well. That includes harassment, unnecessary swearing, sending random links, etc. Message a staff member with solid evidence, such as screenshots, videos, etc, if you feel someone should be punished for something they said through Private Messages.",
+
+    "We, The TXDPSRP Network also strictly follow the Discord Terms of Service (TOS) and their Guidelines! If you are found breaking either of these you will be BANNED without notice and/or explanation"
```
Comment (Ehbw) [inline rules.json:131]:
```suggestion
    "We, The TXNRP Network also strictly follow the Discord Terms of Service (TOS) and their Guidelines! If you are found breaking either of these you will be BANNED without notice and/or explanation"
```

File: rules.json [hunk 2]
```diff
@@ -1,17 +1,170 @@
...
+        "title": "Miscellaneous Rules",
+        "isChild": false,
         "rules": [
-            "You must realistically roleplay at all times. Fail-RolePlay (FailRP), Random Deathmatches (RDM), Vehicle Deathmatches (VDM), Meta-Gaming, Noclipping, Teleporting, Godmode, Invisibility and/or any type of so-called “Trolling” or “No Intent to RP (NITRP)” will not be tolerated.",
-            "Do NOT intentionally bait Law Enforcement Officers, also known as Copbaiting; for example, do NOT try to intentionally get a Law Enforcement Officer to initiate a traffic stop on you.",
-            "Do NOT interfere with any roleplay that you are not apart of.",
-            "Do NOT perform any Erotic RP. This includes roleplay containing sexism and nudity.",
-            "Whatever else would go here, this is just an example...."
+     "• Do NOT sell or purchase anything of some sort that displays the TXNRP Logo, unless approved, distributed, and/or promoted by Head Administration.",
+     "• Do NOT create any discord/teamspeak servers in the name of TXNRP, unless approved by the founder.",
```
Comment (Ehbw) [inline rules.json:148]: We no longer use teamspeak in any capacity 
```suggestion
     "• Do NOT create any discord servers in the name of TXNRP, unless approved by the founder.",
```

File: rules.json [hunk 3]
```diff
@@ -1,17 +1,170 @@
...
+     "•  Do NOT distribute any files or documents which may be harmful to a user.",
+     " • Do NOT distribute or plagiarize any documents created within The TXNRP Network.",
```
Comment (Ehbw) [inline rules.json:150]: Random space
```suggestion
     "• Do NOT distribute or plagiarize any documents created within The TXNRP Network.",
```

File: rules.json [hunk 4]
```diff
@@ -1,17 +1,170 @@
...
+         "• If you believe a forum member has broken the rules, click the report button on the comment, and a staff member will take care of it, if necessary.",
+         "• If you discover a bug, do NOT abuse/exploit it as you will be severely punished. Instead, please file a bug report here so we can fix it as soon as possible.",
+         "• Do NOT have any inappropriate or NSFW profile pictures.",
+         "• If you believe there is a problem with a specific staff member, fill out the Staff Report form located here, and we will take care of the issue, if necessary.",
+         "• Do NOT make any false reports, such as player reports, staff reports or bug reports. This includes using false and/or fabricated evidence/information.",
+
+         "(Rules about applications)",
+         "• Do NOT ask department recruiters, department command staff or staff members to review your application.",
+         "• Do NOT send an application to a department if you already have an application pending for that department.",
+         "• Do NOT lie or make false statements in applications, such as lying about your age or previous TXNRP experience/history.",
```
Comment (Ehbw) [inline rules.json:89]: Slight reword
```suggestion
         "• Do NOT lie or make false statements in applications, such as lying about your age or previous experiences/history in this server.",
```

File: rules.json [hunk 5]
```diff
@@ -1,17 +1,170 @@
...
+        ]
+    },
+    {
+        "title": "Forum Rules",
+        "isChild": false,
+        "rules": [
+         "• Respect and treat everyone EQUALLY. This means to treat and respect both members and the staff team how you would like to be treated.",
+         "• Maturity must be maintained at ALL TIMES.",
+         "• Do not impersonate ANY other members on the Forums. This also applies to any known YouTubers/Celebrities. This falls under “Intentionally Deceiving,” and you can be punished for it.",
+         " • Do not reply to ANY staff related questions. Wait until a staff member responds to that question.",
```
Comment (Ehbw) [inline rules.json:78]:
```suggestion
         "• Do not reply to ANY staff related questions. Wait until a staff member responds to that question.",
```

File: rules.json [hunk 6]
```diff
@@ -1,17 +1,170 @@
...
+         "• Do NOT make any false reports, such as player reports, staff reports or bug reports. This includes using false and/or fabricated evidence/information.",
+
+         "(Rules about applications)",
+         "• Do NOT ask department recruiters, department command staff or staff members to review your application.",
+         "• Do NOT send an application to a department if you already have an application pending for that department.",
+         "• Do NOT lie or make false statements in applications, such as lying about your age or previous TXNRP experience/history.",
+
+            "Staff report: https://txnrp.com/r/staffReport",
+            "Player report: https://txnrp.com/r/report",
+            "Bug report: Link https://txnrp.com/r/bug"
```
Comment (Ehbw) [inline rules.json:93]:
```suggestion
            "Bug report: https://txnrp.com/r/bug"
```

File: rules.json [hunk 7]
```diff
@@ -1,17 +1,170 @@
...
+    "• Obey all staff members. If you feel like you are being treated unfairly by a staff member, please fill out a Staff Report on our forums. Do not complain about it in public.",
+    "• Do not impersonate ANY other members on the Discord Server. This also applies to any known YouTubers/Celebrities. This falls under “Intentionally Deceiving,” and you can be punished for it.",
+    "• Do NOT tag (using @ to ping or send a notification to a specific member) any staff members unless you have a valid reason to do so. Also, do not tag any specific users repeatedly.",
+    "• It is under your discretion to share your own personal information with other members such as race, sex, sexual orientation, financial status, full names, addresses, IP’s, appearances, and voices. However, do NOT share others’ personal information, unless given permission.",
+    "• For the sake of everyone, it is only fair and respectful that TXNRP is an ENGLISH only network. Please refrain from speaking any other languages than English in any text/voice channels.",
+    "Advertising",
+    "• Do NOT have a social media link, IP, or another FiveM server in your discord status message.",
```
Comment (Ehbw) [inline rules.json:125]: @IceTheDev do we still plan to enforce this?

**Review by Ehbw — CHANGES_REQUESTED** [review]

File: rules.json (hunk 7 above)
Comment (IceTheDev) [reply rules.json:125]: don't know yet, still need to go through the rules and re-write them. A lot of them are confusing and some just don't make any sense.

**Pull request closed without merging**